solution/infra/lambda.tf

```diff
@@ -0,0 +1,88 @@
+locals {
+  file_name = "../artifacts/cloud-backend-test-lambda.zip"
+  mapbox_api_key = "INSERT_YOUR_KEY_HERE"
```

Comment:
Do you think this variable could be somewhere more organized?

Reply:
I guess a little of the answer is in your question 😁, it would be better to use a variable instead of a local. I could declare it in the "variables.tf" file and maybe use a ".tfvars" file to specify its value.